services/frontend/Makefile

```diff
@@ -0,0 +1,3 @@
+frontend: $(GOVUK_ROOT_DIR)/frontend content-store router static govuk-content-schemas
+	$(COMPOSE_RUN) frontend-lite bundle
```

Comment:
Like [calendars](https://github.com/alphagov/govuk-docker/blob/753eece40fbc7b326c4be65f4be9aee42a07c868/services/calendars/Makefile#L3), I think we'll need a step here to publish the special routes and need a setup stack

Reply:
Done